### PR TITLE
[Dungeon] Uldaman

### PR DESCRIPTION
**Pull request by Knindzagxg**
Changes:

Min, Max & Modifier Damage for whole Dungeon.
Added flags for each boss, so they will appear with skull instead regular level elite.

All checks have been done trough Classic Realms and Youtube videos.

How to test:
Apply SQL
Restart Server and delete your Cache.

Make sure to add very very basic gear towards each Dungeon.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1588337806789523700.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 269, `maxdmg` = 346, `DamageModifier` = 1.01 WHERE `entry` = 6906;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 193, `maxdmg` = 249, `DamageModifier` = 1.01 WHERE `entry` = 6907;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 215, `maxdmg` = 276, `DamageModifier` = 1.01 WHERE `entry` = 6908;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 269, `maxdmg` = 346, `DamageModifier` = 1.01 WHERE `entry` = 7206;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 204, `maxdmg` = 295, `DamageModifier` = 1.01 WHERE `entry` = 7291;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 251, `maxdmg` = 325, `DamageModifier` = 1.01 WHERE `entry` = 4854;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 269, `maxdmg` = 346, `DamageModifier` = 1.01 WHERE `entry` = 7023;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 263, `maxdmg` = 361, `DamageModifier` = 1.01 WHERE `entry` = 7228;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 329, `maxdmg` = 364, `DamageModifier` = 1.01 WHERE `entry` = 2748;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1588337806789523700.sql:50]: newline
```suggestion
UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 329, `maxdmg` = 364, `DamageModifier` = 1.01 WHERE `entry` = 2748;

```

**Comment by Geekena** [comment]
damage and spells are ok
hardest part was to find object to start event :)

**Comment by Helias** [comment]
Thank you @Geekena for testing several PRs

**Comment by Kitzunu** [comment]
Ty for the PR!

**Pull request merged**